landing_page_backend/contacts/management/commands/weekly_reports.py

```diff
@@ -0,0 +1,22 @@
+from django.core.mail import EmailMessage
+from django.core.management.base import BaseCommand
+
+from contacts.models import Contact
+from contacts.utils import write_data_to_sfv_file
+from landing_page_backend.settings import EMAIL_HOST_USER
+
+
+class Command(BaseCommand):
+    help = 'Weekly reports'
```

Comment:
I would write something more verbose, like:
Exports previous week contact to a file and sends an email with a report file attached to an administrator.